### PR TITLE
switch to using tokio-tun to eliminate dependency on launcher

### PR DESCRIPTION
**Pull request by cpacejo**

### CODE REVIEW

File: adapter/ph/src/main.rs [hunk 1]
```diff
@@ -292,9 +258,8 @@ fn main() -> ExitCode {
                 op_outq,
             ));
 
-            js.spawn(rpc_worker::launch(&*asm, &*unix_socket));
```
Comment (cpacejo) [inline adapter/ph/src/main.rs:295]: Drive-by fix: this is a duplicate of line 243/202.

**Review by owenrhysthomas — APPROVED** [review]
Cool! Just make sure to format before merging.
